sys-fs/udisks/udisks-2.9.4-r1.ebuild

```diff
@@ -97,8 +97,8 @@ src_configure() {
 		--with-html-dir="${EPREFIX}"/usr/share/gtk-doc/html
 		--with-modprobedir="${EPREFIX}"/lib/modprobe.d
 		--with-systemdsystemunitdir="$(systemd_get_systemunitdir)"
-		--with-tmpfilesdir="/usr/lib/tmpfiles.d"
-		--with-udevdir="$(get_udevdir)"
+		--with-tmpfilesdir="${EPREFIX}"/usr/lib/tmpfiles.d
+		--with-udevdir="$EPREFIX$(get_udevdir)"
```

Comment:
Please include braces for consistency: `${EPREFIX}`

Reply:
done